docs/data/date-pickers/migration-lab/migration-lab.md

```diff
@@ -0,0 +1,85 @@
+---
+title: MUI X - Migration from the lab
+---
+
+# MUI X - Migration from the lab
+
+<p class="description">MUI Date and time pickers are now available on X!</p>
+
+## Introduction
+
+This is a reference for migrating your site's pickers from `@mui/lab` to `@mui/x-date-pickers` or `@mui/x-date-pickers-pro`.
+This migration should be easy
+
+We explored the reasons of this migration in a [blog post](https://mui.com/blog/lab-date-pickers-to-mui-x/)
+
+## License
+
+Most of our components remains MIT and are accessible for free in `@mui/x-date-pickers`.
+
+The components `DateRangePicker`, `DateRangePickerDay`, `DesktopDateRangePicker`, `MobileDateRangePicker` and `StaticDateRangePicker`
+were marked as "intended for MUI X Pro" in our documentation and are now part of MUI X Pro.
+
+If you are using one of these components, you will have to take a Pro license in order to migrate to `@mui/x-date-pickers-pro` (see the [Pricing](https://mui.com/pricing/) page for more information).
+
+**Note**: If you already have a license for `@mui/x-data-grid-pro`, you can use the same one for `@mui/x-date-pickers-pro` with no additional fee!
+
+## How to migrate ?
+
+### Installation
+
+#### Pro Plan
```

Comment:
I think it should be step by step with 
```
## Migration steps

### 1. Install MUI X package

...installation

> add a note about license key for pro plan

### 2. Run the code mod

...We have prepared a codemod to help you migrate your codebase

which will transform the imports like this:
...example
```

Reply:
The main package being different between pro and community I found it easier to split the `sh` code entirely between the plans.

Would the following structure work for you ?

```suggestion
## Migration steps

### 1. Install MUI X packages


```sh
// with npm
npm install @mui/x-date-pickers-pro @mui/x-license-pro

// with yarn
yarn add @mui/x-date-pickers-pro @mui/x-license-pro
```

When you purchase a commercial license, you'll receive a license key by email.
You must set the license key before rendering the first component.

```jsx
import { LicenseInfo } from '@mui/x-license-pro';

LicenseInfo.setLicenseKey(
  'x0jTPl0USVkVZV0SsMjM1kDNyADM5cjM2ETPZJVSQhVRsIDN0YTM6IVREJ1T0b9586ef25c9853decfa7709eee27a1e',
);
```

More information [here](/x/advanced-components/#license-key-installation)

#### Community Plan

```sh
// with npm
npm install @mui/x-date-pickers

// with yarn
yarn add @mui/x-data-grid
```

### 2. Run the code mod

We have prepared a codemod to help you migrate your codebase

```sh
npx @mui/codemod v5.0.0/date-pickers-moved-to-x <path>
```

Which will transform the imports like this:

```ts
// before
import DatePicker from '@mui/lab/DatePicker';
import DateRangePicker from '@mui/lab/DateRangePicker';
import { DatePicker, DateRangePicker } from '@mui/lab';

// after
import DatePicker from '@mui/x-date-pickers/DatePicker';
import DateRangePicker from '@mui/x-date-pickers-pro/DateRangePicker';
import { DatePicker } from '@mui/x-date-pickers'; // DatePicker is also available in `@mui/x-date-pickers-pro`
import { DateRangePicker } from '@mui/x-date-pickers-pro';
```
```

Reply:
Would it make sense to bring the **Community plan** before the **Pro**? I'll let you decide on this.

```
## Migration steps

### 1. Install MUI X packages

#### Community Plan
...

#### Pro plan
...

### 2. Run the code mod
...
```

Reply:
For the doc Sam wanted to put the pro first to give visibility
But for the migration, it's probably best to put the most used first indeed

Reply:
Done